### PR TITLE
Resolves #1812

### LINKED ISSUE #1812: A guide about how to set parameters such as managedRuntimeVersion on New-WebAppPool
Is it possible to add an example or link about how to set parameters on the application pool created? It's very likely people need to do this if they are trying to create an application pool.
For example, `managedRuntimeVersion`, `autoStart` etc..

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8a01808e-fcdb-8533-0047-6906fd13db39
* Version Independent ID: 203ea1cb-1ca4-ed9a-78cf-1ea1d593e6de
* Content: [New-WebAppPool (webadminstration)](https://docs.microsoft.com/en-us/powershell/module/webadminstration/new-webapppool?view=winserver2012-ps#feedback)
* Content Source: [docset/winserver2012-ps/webadminstration/New-WebAppPool.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/live/docset/winserver2012-ps/webadminstration/New-WebAppPool.md)
* Product: **w10**
* Technology: **powershell**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Provide more info on setting Application pool properties
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1812

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/webadministration/New-WebAppPool.md [hunk 1]
```diff
@@ -30,17 +30,34 @@ New-WebAppPool [-Name] <String> [-Force] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **New-WebAppPool** cmdlet creates an Internet Information Services (IIS) application pool.
+The **New-WebAppPool** cmdlet creates an Internet Information Services (IIS) application pool. For changing different properties of the application pool after creation check the following article - [PowerShell Snap-in: Making Configuration Changes to Websites and App Pools](https://docs.microsoft.com/iis/manage/powershell/powershell-snap-in-making-simple-configuration-changes-to-web-sites-and-application-pools).
```
Comment (dariomws) [inline docset/windows/webadministration/New-WebAppPool.md:33]:
```suggestion
The **New-WebAppPool** cmdlet creates an Internet Information Services (IIS) application pool. For changing different properties of the application pool after creation, see [PowerShell Snap-in: Making Configuration Changes to Websites and App Pools](https://docs.microsoft.com/iis/manage/powershell/powershell-snap-in-making-simple-configuration-changes-to-web-sites-and-application-pools).
```

**Review by dariomws — APPROVED** [review]
Just a small suggestion.

**Review by get-itips — COMMENTED** [review]
Agree when @dariomws suggestions are applied.

**Review by get-itips — APPROVED** [review]

File: docset/windows/webadministration/New-WebAppPool.md [hunk 2]
```diff
@@ -30,17 +30,34 @@ New-WebAppPool [-Name] <String> [-Force] [<CommonParameters>]
...
+### Example 2: Create an IIS application pool and set autoStart
+```powershell
+IIS:\> $newAppPool = New-WebAppPool -Name "NewAppPool"
+IIS:\> $newAppPool.autoStart = "false"
+IIS:\> $newAppPool | Set-Item
+```
+
+This command creates an IIS application pool named NewAppPool and sets **autoStart** property to false.
+
+### Example 2: Create an IIS application pool and set managedRuntimeVersion
```
Comment (JohanFreelancer9) [inline docset/windows/webadministration/New-WebAppPool.md:53]:
```suggestion
### Example 3: Create an IIS application pool and set managedRuntimeVersion
```

File: docset/winserver2012-ps/webadminstration/New-WebAppPool.md [hunk 3]
```diff
@@ -21,16 +21,33 @@ New-WebAppPool [-Name] <String> [-Force] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-Creates a new IIS application pool.
+The **New-WebAppPool** cmdlet creates an Internet Information Services (IIS) application pool. For changing different properties of the application pool after creation check, see [PowerShell Snap-in: Making Configuration Changes to Websites and App Pools](https://docs.microsoft.com/iis/manage/powershell/powershell-snap-in-making-simple-configuration-changes-to-web-sites-and-application-pools).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/webadminstration/New-WebAppPool.md:24]:
```suggestion
The **New-WebAppPool** cmdlet creates an Internet Information Services (IIS) application pool. For changing different properties of the application pool after creation, see [PowerShell Snap-in: Making Configuration Changes to Websites and App Pools](https://docs.microsoft.com/iis/manage/powershell/powershell-snap-in-making-simple-configuration-changes-to-web-sites-and-application-pools).
```

File: docset/winserver2012-ps/webadminstration/New-WebAppPool.md [hunk 4]
```diff
@@ -21,16 +21,33 @@ New-WebAppPool [-Name] <String> [-Force] [<CommonParameters>]
...
+```powershell
+IIS:\> $newAppPool = New-WebAppPool -Name "NewAppPool"
+IIS:\> $newAppPool.autoStart = "false"
+IIS:\> $newAppPool | Set-Item
 ```
-IIS:\>New-WebAppPool NewAppPool
+
+This command creates an IIS application pool named NewAppPool and sets **autoStart** property to false.
+
+### Example 2: Create an IIS application pool and set managedRuntimeVersion
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/webadminstration/New-WebAppPool.md:44]:
```suggestion
### Example 3: Create an IIS application pool and set managedRuntimeVersion
```

File: docset/winserver2012r2-ps/webadministration/New-WebAppPool.md [hunk 5]
```diff
@@ -29,17 +29,34 @@ New-WebAppPool [-Name] <String> [-Force] [<CommonParameters>]
...
+### Example 2: Create an IIS application pool and set autoStart
+```powershell
+IIS:\> $newAppPool = New-WebAppPool -Name "NewAppPool"
+IIS:\> $newAppPool.autoStart = "false"
+IIS:\> $newAppPool | Set-Item
+```
+
+This command creates an IIS application pool named NewAppPool and sets **autoStart** property to false.
+
+### Example 2: Create an IIS application pool and set managedRuntimeVersion
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/webadministration/New-WebAppPool.md:52]:
```suggestion
### Example 3: Create an IIS application pool and set managedRuntimeVersion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]
Approve pending changes

**Pull request merged**